activitychart-basic.html

```diff
@@ -106,29 +106,57 @@
       },
 
       _showLabel: function (d, i) {
```

Comment:
I'm against that here. It's a D3 thing to use this signature; d being the datum and i being its index.